<launch>
    <node name="stopbar_detector" pkg="rr_iarrc" type="stopbar_detector" output="screen" required="true">
        <!-- Output Topics -->
        <param name="stopbar_near" type="string" value="/stopbar_near"/>
        <param name="stopbar_angle" type="string" value="/stopbar_angle"/>

        <!-- Camera Params-->
        <param name="overhead_image_subscription" type="string" value="/camera_center/lines/detection_img_transformed" />
        <param name="pixels_per_meter" type="int" value="50" />

        <!-- Stop Bar Detector Params -->
        <param name="stopBarGoalAngle" type="double" value="0.0" /> <!-- angle in degrees -->
        <param name="stopBarGoalAngleRange" type="double" value="25.0" /> <!-- we can come at the line from many angles! -->
        <param name="stopBarTriggerDistance" type="double" value="3.5" /> <!-- distance in meters -->
        <param name="sleep_adjust" type="double" value=".325"/>

        <!-- Hough Line Params-->
        <param name="houghThreshold" type="int" value="65" /> <!-- see opencv HoughLinesP for more info -->
        <param name="houghMinLineLength" type="double" value="50" />
        <param name="houghMaxLineGap" type="double" value="10" />
    </node>
</launch>
